src/sections/trade/sections/otc/orders/OtcOrdersData.tsx

```diff
@@ -1,45 +1,144 @@
 import BN from "bignumber.js"
 import { AssetLogo } from "components/AssetIcon/AssetIcon"
 import { MultipleIcons } from "components/MultipleIcons/MultipleIcons"
+import { Icon } from "components/Icon/Icon"
 import { Text } from "components/Typography/Text/Text"
 import { useTranslation } from "react-i18next"
 import { OfferingPair } from "./OtcOrdersData.utils"
+import { motion } from "framer-motion"
+import { useRpcProvider } from "providers/rpcProvider"
+
+function abbreviateNumber(price: BN): string {
```

Comment:
We don't have this elsewhere? Would imagine it would be somewhere in a lib or helpers

Reply:
Could not find an instance of it so moved it to helpers for now.